catalyst-gateway/bin/src/db/index/schema/cql/namespace.cql

```diff
@@ -1,4 +1,6 @@
 -- Create the namespace in the DB.
 -- Template.
-CREATE KEYSPACE IF NOT EXISTS {{keyspace}}
-    With replication = {'class': 'NetworkTopologyStrategy','replication_factor': 1};
\ No newline at end of file
+CREATE KEYSPACE IF NOT EXISTS {{keyspace}} WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', '{{region1}}' : {{replication_factor}}, '{{region2}}' : {{replication_factor}} };
```

Comment:
```suggestion
CREATE KEYSPACE IF NOT EXISTS {{keyspace}} WITH {{options}};
```

Instead of busting out everything after REPLICATION into individual variables,
I suggest instead to have a `CASSANCE_{name}_KEYSPACE_OPTIONS` env var.
Default value is `REPLICATION = {'class': 'NetworkTopologyStrategy','replication_factor': 1}`;
SRE can set it to:
`REPLICATION = { 'class' : 'NetworkTopologyStrategy', 'eu_central_1' : 3, 'eu_west_1' : 3 }`
or whatever they need.  That way if there needs to be further options, more zones, different replication factors per zone, whatever, its just a setting they control.

And then remove those individual options.

For example: 
Cassandra also supports a "DURABLE_WRITES" option.  What if we want that?
Scylla also supports a "TABLETS" option, which we may wish to change.
AWS Keyspaces also supports "TAGS" option, which SRE may need to set.
AWS also allows just `REPLICATION = {'class': 'SingleRegionStrategy'}` which may be the best option for our use case.

By generalizing OPTIONS we allow all of this, and we don't need to keep adding features to support different cassandra instance types.